game/scripts/vscripts/items/item_greater_tranquil_boots.lua

```diff
@@ -0,0 +1,261 @@
+item_greater_tranquil_boots = class({})
+
+LinkLuaModifier( "modifier_item_greater_tranquil_boots", "items/item_greater_tranquil_boots.lua", LUA_MODIFIER_MOTION_NONE )
+LinkLuaModifier( "modifier_item_greater_tranquil_boots_sap", "items/item_greater_tranquil_boots.lua", LUA_MODIFIER_MOTION_NONE )
+
+--------------------------------------------------------------------------------
+
+function item_greater_tranquil_boots:GetAbilityTextureName()
+	local baseName = self.BaseClass.GetAbilityTextureName( self )
+	
+	brokeName = ""
+	
+	if self.tranqMod and not self.tranqMod:IsNull() and self.tranqMod:GetRemainingTime() > 0 then
+		brokeName = "_active"
+	end
+	
+	return baseName .. brokeName
+end
+
+--------------------------------------------------------------------------------
+
+function item_greater_tranquil_boots:GetIntrinsicModifierName()
+	return "modifier_item_greater_tranquil_boots"
+end
+
+--------------------------------------------------------------------------------
+
+modifier_item_greater_tranquil_boots = class({})
+
+--------------------------------------------------------------------------------
+
+function modifier_item_greater_tranquil_boots:IsHidden()
+	return true
+end
+
+function modifier_item_greater_tranquil_boots:IsDebuff()
+	return false
+end
+
+function modifier_item_greater_tranquil_boots:IsPurgable()
+	return false
+end
+
+function modifier_item_greater_tranquil_boots:GetAttributes()
+	return MODIFIER_ATTRIBUTE_MULTIPLE
+end
+
+function modifier_item_greater_tranquil_boots:DestroyOnExpire()
+	return false
+end
+
+--------------------------------------------------------------------------------
+
+function modifier_item_greater_tranquil_boots:OnCreated( event )
+	local spell = self:GetAbility()
+		
+	spell.tranqMod = self
+		
+	if IsServer() then
+		self:SetDuration( spell:GetCooldownTime(), true )
+			
+		self:StartIntervalThink( 0.1 )
+	end
+end
+
+--------------------------------------------------------------------------------
+
+if IsServer() then
+	function modifier_item_greater_tranquil_boots:OnIntervalThink()
+		if self.storedDamage and self.storedDamage > 0 then
+			local parent = self:GetParent()
+			local spell = self:GetAbility()
+			local maxHeal = math.min( spell:GetSpecialValueFor( "regen_from_creeps" ) * 0.1, self.storedDamage )
+			
+			parent:Heal( maxHeal, parent )
+			
+			self.storedDamage = self.storedDamage - maxHeal
```

Comment:
You missed the line right above it - it heals for either 10% of regen_from_creeps or the remainder of stored damage, whichever is lower.